resources/views/checkout/partials/sidebar/summary.blade.php

```diff
@@ -1,7 +1,7 @@
 <x-rapidez-ct::card class="relative">
-    <div class="absolute inset-x-0 bottom-full -translate-y-6 max-md:hidden [&>*]:h-auto [&>*]:max-h-20 [&>*]:w-full [&>*]:object-contain">
+    <a href="/" class="absolute inset-x-0 bottom-full -translate-y-6 max-md:hidden [&>*]:h-auto [&>*]:max-h-20 [&>*]:w-full [&>*]:object-contain">
```

Comment:
i don't know if i want the logo in the checkout to be clickable sending the user away, what do you think?